src/app/profile/yourGiving/givingRecipientView/donationRecipient/donationRecipient.tpl.html

```diff
@@ -0,0 +1,88 @@
+<div class="panel panel-margin-bottom recent-recipient-row">
+  <div class="panel-body">
+
+    <!-- Recipient Row -->
+    <div class="row">
+      <div class="col-md-2 col-full-height col-sm-12 col-xs-12 mb">
+        <img class="full-width" desig-src="{{$ctrl.recipient['designation-number']}}">
+      </div>
+      <div class="col-sm-12 col-md-7 col-full-height">
+        <h4 class="mb0">{{$ctrl.recipient['designation-name']}}</h4>
+        <small>#{{$ctrl.recipient['designation-number']}}</small>
+
+        <div class="row">
+          <div class="col-sm-3 col-md-3">
+            <div class="form-group mb0">
+              <label translate>Year-to-date:</label>
+              <ul class="list-unstyled list-recurring">
+                <li>
+                  <h4>{{$ctrl.recipient['year-to-date-amount'] | currency:'$'}}</h4>
+                  <span class="meta">{{$ctrl.currentDate | date:'mediumDate'}}</span>
+                </li>
+              </ul>
+            </div>
+          </div>
+          <div class="col-sm-3 col-md-3">
+            <div class="form-group mb0" ng-if="$ctrl.recipient['mostrecentdonation']">
+              <label translate>Most Recent:</label>
+              <ul class="list-unstyled list-recurring">
+                <li>
+                  <h4>{{$ctrl.recipient['mostrecentdonation']['historical-donation-line']['amount'] | currency:'$'}}</h4>
+                  <span class="meta">{{$ctrl.recipient['mostrecentdonation']['historical-donation-line']['transaction-date']['display-value'] | date:'mediumDate'}}</span>
+                </li>
+              </ul>
+            </div>
+          </div>
+          <div class="col-sm-4 col-md-4">
+            <div class="form-group mb0" ng-if="$ctrl.recipient['mostrecentdonationRecurringdonationelement']">
+              <label translate>Recurring Gifts:</label>
+              <ul class="list-unstyled list-recurring">
+                <li ng-repeat="recurring in $ctrl.recipient['mostrecentdonationRecurringdonationelement']">
```

Comment:
My account has data from the old prod site that must have been imported a year or two ago.  For some reason I have lots of these recurring gifts of the same number and date. Idk if that is an issue with it being old imported data or the new backend implementation.

![screen shot 2016-10-03 at 10 56 10 am](https://cloud.githubusercontent.com/assets/756501/19047763/0a41480c-8958-11e6-8c39-011f69468b71.png)

```json
"_recurringdonationelement": [
    {
        "self": {
            "type": "donations.recurring",
            "uri": "/donations/recurring/crugive/gewuuwktkbefq=",
            "href": "https://cortex-gateway-stage.cru.org/cortex/donations/recurring/crugive/gewuuwktkbefq="
        },
        "links": [
            {
                "rel": "paymentmethods",
                "uri": "/paymentmethods/crugive",
                "href": "https://cortex-gateway-stage.cru.org/cortex/paymentmethods/crugive"
            },
            {
                "rel": "paymentmethod",
                "uri": "/paymentmethods/crugive/m5uxszddnv5hsz3fhu=",
                "href": "https://cortex-gateway-stage.cru.org/cortex/paymentmethods/crugive/m5uxszddnv5hsz3fhu="
            }
        ],
        "donation-lines": [
            {
                "amount": 15,
                "designation-name": "Test Person (1234567)",
                "designation-number": "1234567",
                "donation-line-row-id": "1-JYSPHZ",
                "donation-line-status": "Standard",
                "payment-method-id": "giydcmzygf=",
                "updated-donation-line-status": "",
                "updated-payment-method-id": "",
                "updated-rate": {
                    "recurrence": {
                        "interval": ""
                    }
                },
                "updated-recurring-day-of-month": "",
                "updated-start-month": "",
                "updated-start-year": ""
            }
        ],
        "donation-row-id": "1-JYSPHX",
        "donation-status": "Active",
        "effective-status": "Active",
        "next-draw-date": {
            "display-value": "2016-10-20",
            "value": 1476921600000
        },
        "rate": {
            "recurrence": {
                "interval": "Monthly"
            }
        },
        "recurring-day-of-month": "20",
        "start-date": {
            "display-value": "2016-09-30",
            "value": 1475193600000
        }
    },
    {
        "self": {
            "type": "donations.recurring",
            "uri": "/donations/recurring/crugive/gewuuwsjgnkue=",
            "href": "https://cortex-gateway-stage.cru.org/cortex/donations/recurring/crugive/gewuuwsjgnkue="
        },
        "links": [
            {
                "rel": "paymentmethods",
                "uri": "/paymentmethods/crugive",
                "href": "https://cortex-gateway-stage.cru.org/cortex/paymentmethods/crugive"
            },
            {
                "rel": "paymentmethod",
                "uri": "/paymentmethods/crugive/m5uxszddnv5hsz3fhu=",
                "href": "https://cortex-gateway-stage.cru.org/cortex/paymentmethods/crugive/m5uxszddnv5hsz3fhu="
            }
        ],
        "donation-lines": [
            {
                "amount": 15,
                "designation-name": "Test Person (1234567)",
                "designation-number": "1234567",
                "donation-line-row-id": "1-JZI3UD",
                "donation-line-status": "Standard",
                "payment-method-id": "giydcmzyge=",
                "updated-donation-line-status": "",
                "updated-payment-method-id": "",
                "updated-rate": {
                    "recurrence": {
                        "interval": ""
                    }
                },
                "updated-recurring-day-of-month": "",
                "updated-start-month": "",
                "updated-start-year": ""
            }
        ],
        "donation-row-id": "1-JZI3UB",
        "donation-status": "Active",
        "effective-status": "Active",
        "next-draw-date": {
            "display-value": "2016-10-20",
            "value": 1476921600000
        },
        "rate": {
            "recurrence": {
                "interval": "Monthly"
            }
        },
        "recurring-day-of-month": "20",
        "start-date": {
            "display-value": "2016-09-30",
            "value": 1475193600000
        }
    },
    ...
]
```